src/components/FilmsList.vue

```diff
@@ -83,7 +80,9 @@ export default {
     ...mapActions(["getMovies", "getMovieDetails", "nextMoviesPage"]),
   },
   created() {
-    this.fetchFilms();
+    if (!this.searchQuery) {
```

Comment:
Is it ever possible for `searchQuery` to not be false when `created` runs?

Reply:
When we make a request from another page, the user is redirected to the main page, then without this condition, the request starts first and then it is overwritten by the method from created